### PR TITLE
Fix for Julia master

### PR DESCRIPTION
**Pull request by simonster**

### CODE REVIEW

File: src/LibExpat.jl [hunk 1]
```diff
@@ -15,7 +15,6 @@ include("lX_defines_h.jl")
 include("lX_expat_h.jl")
 #include("lX_exports_h.jl")
 
-@c Ptr{XML_LChar} XML_ErrorString (Cint,) libexpat
```
Comment (tkelman) [inline src/LibExpat.jl]: why is this removed?

Comment (simonster) [inline src/LibExpat.jl]: It's already defined in https://github.com/amitmurthy/LibExpat.jl/blob/882d8fc8e8ac899aaffa85b8022a849651c6fd77/src/lX_expat_h.jl#L66, so it produces a method redefinition warning. However, upon further inspection, maybe the other method needs to be changed to take `Cint` instead of `Void`?

**Comment by vtjnash** [comment]
might want to fix / delete the signatures with `size_t` also?

**Comment by simonster** [comment]
Might also want to fix the [generator code](https://github.com/amitmurthy/LibExpat.jl/blob/master/gen/generate.jl), but Clang.jl isn't building on my system.

**Comment by tkelman** [comment]
@amitmurthy could you enable attobot here? makes release tagging easier

**Comment by amitmurthy** [comment]
Done.
